src/commands/ExecCommand.js

```diff
@@ -18,17 +18,19 @@ export const builder = {
     group: "Command Options:",
     describe: "Bail on exec execution when the command fails within a package",
     type: "boolean",
-    default: true,
+    default: undefined,
```

Comment:
@evocateur I was just raising this issue, and in describing the rationale, I thought of a possible solution. What if instead of merging the durable options *after* yargs parses the command, we read them *before* and pass them in as the yargs options defaults?